pkg/flowaggregator/flowaggregator.go

```diff
@@ -325,50 +344,280 @@ func (fa *flowAggregator) initExportingProcess() error {
 }
 
 func (fa *flowAggregator) Run(stopCh <-chan struct{}) {
-	exportTicker := time.NewTicker(fa.exportInterval)
-	defer exportTicker.Stop()
 	go fa.collectingProcess.Start()
 	defer fa.collectingProcess.Stop()
 	go fa.aggregationProcess.Start()
 	defer fa.aggregationProcess.Stop()
+	go fa.flowRecordExpiryCheck(true, stopCh)
+	go fa.flowRecordExpiryCheck(false, stopCh)
+
+	<-stopCh
+}
+
+func (fa *flowAggregator) flowRecordExpiryCheck(isActiveTimeout bool, stopCh <-chan struct{}) {
+	var timeout time.Duration
+	var updatePQ ipfixintermediate.FlowKeyRecordMapCallBack
```

Comment:
`updatePQ` is a function, not a priority queue. I recommend making it clear in the name. Maybe name it `updatePriorityQueueFn`